tests/http_tests.cpp

```diff
@@ -1,73 +1,79 @@
 #include <gtest/gtest.h>
-#include <HTTP/Parser.h>
+#include <HTTP/HTTPRequest.h>
 
 TEST(first_test, dummy_test){
     ASSERT_FALSE(false);
 };
 TEST(HTTPParser, RequestURI_valid){
-    HTTP::Parser parser;
+    HTTP::HTTPRequest parser;
     std::string pom = "*";
```

Comment:
Name your variables based on the terms of the subject area, so that the variable name clearly describes its purpose. Example: "test_uri" or "uri".